diff --git a/content/renderer/render_thread_impl.h b/content/renderer/render_thread_impl.h
index 57e3c329c2..c51b9d78bb 100644
--- a/content/renderer/render_thread_impl.h
+++ b/content/renderer/render_thread_impl.h
@@ -408,6 +408,8 @@ class CONTENT_EXPORT RenderThreadImpl
       blink::mojom::OriginTrialsSettingsPtr origin_trial_settings,
       blink::mojom::PerformanceTier cpu_performance_tier,
       uint64_t trace_id) override;
+  void SetUxrConfig(const base::flat_map<std::string, std::string>& config)
+      override;
 #if BUILDFLAG(IS_MAC)
   void UpdateScrollbarTheme(
       mojom::UpdateScrollbarThemeParamsPtr params) override;
