diff --git a/content/renderer/renderer_main.cc b/content/renderer/renderer_main.cc
index 8ab1d66318..9ee8e09069 100644
--- a/content/renderer/renderer_main.cc
+++ b/content/renderer/renderer_main.cc
@@ -90,6 +90,13 @@
 #include "mojo/public/cpp/bindings/lib/test_random_mojo_delays.h"
 #endif
 
+#include "third_party/icu/source/i18n/unicode/timezone.h"
+#include "third_party/icu/source/common/unicode/unistr.h"
+
+#include <stdlib.h>
+#include <time.h>
+#include <memory>
+
 namespace content {
 namespace {
 
@@ -158,6 +165,7 @@ int RendererMain(MainFunctionParams parameters) {
 
   const base::CommandLine& command_line = *parameters.command_line;
 
+
 #if BUILDFLAG(IS_MAC)
   base::apple::ScopedNSAutoreleasePool* pool = parameters.autorelease_pool;
 #endif  // BUILDFLAG(IS_MAC)
