diff --git a/third_party/blink/renderer/core/frame/navigator.cc b/third_party/blink/renderer/core/frame/navigator.cc
index 583f05f8dd..6163cc8105 100644
--- a/third_party/blink/renderer/core/frame/navigator.cc
+++ b/third_party/blink/renderer/core/frame/navigator.cc
@@ -22,6 +22,7 @@
 
 #include "third_party/blink/renderer/core/frame/navigator.h"
 
+#include "base/uxr_config.h"
 #include "third_party/blink/renderer/bindings/core/v8/script_controller.h"
 #include "third_party/blink/renderer/core/dom/document.h"
 #include "third_party/blink/renderer/core/execution_context/navigator_base.h"
@@ -98,12 +99,13 @@ bool Navigator::cookieEnabled() const {
 }
 
 bool Navigator::webdriver() const {
-  if (RuntimeEnabledFeatures::AutomationControlledEnabled())
-    return true;
-
-  bool automation_enabled = false;
-  probe::ApplyAutomationOverride(GetExecutionContext(), automation_enabled);
-  return automation_enabled;
+  if (base::UxrConfig::GetInstance().Get("uxr-fingerprint-off") != "true")
+    return false;
+  if (RuntimeEnabledFeatures::AutomationControlledEnabled())
+    return true;
+  bool automation_enabled = false;
+  probe::ApplyAutomationOverride(GetExecutionContext(), automation_enabled);
+  return automation_enabled;
 }
 
 String Navigator::GetAcceptLanguages() {
