diff --git a/content/browser/devtools/protocol/network_handler.cc b/content/browser/devtools/protocol/network_handler.cc
index 1b7f400..bd05787 100644
--- a/content/browser/devtools/protocol/network_handler.cc
+++ b/content/browser/devtools/protocol/network_handler.cc
@@ -200,20 +200,22 @@ std::unique_ptr<Network::CookiePartitionKey> BuildCookiePartitionKey(
 
 std::unique_ptr<Network::Cookie> BuildCookie(
     const net::CanonicalCookie& cookie) {
-  DCHECK(cookie.ExpiryDate().is_null() || !cookie.ExpiryDate().is_inf());
+  const bool is_session = cookie.ExpiryDate().is_null() ||
+                          cookie.ExpiryDate().is_inf() ||
+                          !cookie.IsPersistent();
   std::unique_ptr<Network::Cookie> devtools_cookie =
       Network::Cookie::Create()
           .SetName(cookie.Name())
           .SetValue(cookie.Value())
           .SetDomain(cookie.Domain())
           .SetPath(cookie.Path())
-          .SetExpires(cookie.ExpiryDate().is_null()
+          .SetExpires(is_session
                           ? -1
                           : cookie.ExpiryDate().InSecondsFSinceUnixEpoch())
           .SetSize(cookie.Name().length() + cookie.Value().length())
           .SetHttpOnly(cookie.IsHttpOnly())
           .SetSecure(cookie.SecureAttribute())
-          .SetSession(!cookie.IsPersistent())
+          .SetSession(is_session)
           .SetPriority(BuildCookiePriority(cookie.Priority()))
           .SetSourceScheme(BuildCookieSourceScheme(cookie.SourceScheme()))
           .SetSourcePort(cookie.SourcePort())
