diff --git a/v8/src/inspector/v8-runtime-agent-impl.cc b/v8/src/inspector/v8-runtime-agent-impl.cc
index b17432ca819792a6857676ef1ff68e4cdac587b8..9bfdc5ab462ba1c4c4b1fdfa519f4067a0e46d3e 100644
--- a/v8/src/inspector/v8-runtime-agent-impl.cc
+++ b/v8/src/inspector/v8-runtime-agent-impl.cc
@@ -42,6 +42,7 @@
 #include "include/v8-inspector.h"
 #include "include/v8-microtask-queue.h"
 #include "src/debug/debug-interface.h"
+#include "src/flags/flags.h"
 #include "src/inspector/injected-script.h"
 #include "src/inspector/inspected-context.h"
 #include "src/inspector/protocol/Protocol.h"
@@ -1074,6 +1075,7 @@ void V8RuntimeAgentImpl::addBindings(InspectedContext* context) {
   int contextGroupId = context->contextGroupId();
   int contextId = context->contextId();
   const String16 contextName = context->humanReadableName();
+  if (v8::internal::v8_flags.uxr_devtools_runtime_suppression) return;
   if (!m_enabled) return;
 
   protocol::DictionaryValue* globalBindings =
@@ -1229,7 +1231,13 @@ void V8RuntimeAgentImpl::inspect(
 }
 
 void V8RuntimeAgentImpl::messageAdded(V8ConsoleMessage* message) {
-  if (m_enabled) reportMessage(message, true);
+  if (m_enabled && !v8::internal::v8_flags.uxr_devtools_runtime_suppression) {
+    reportMessage(message, true);
+  }
+}
+
+bool V8RuntimeAgentImpl::enabled() const {
+  return m_enabled && !v8::internal::v8_flags.uxr_devtools_runtime_suppression;
 }
 
 bool V8RuntimeAgentImpl::reportMessage(V8ConsoleMessage* message,
