diff --git a/tools/metrics/BUILD.gn b/tools/metrics/BUILD.gn
index 4cf523f9cd..6f992aea7d 100644
--- a/tools/metrics/BUILD.gn
+++ b/tools/metrics/BUILD.gn
@@ -40,6 +40,7 @@
     ":ukm_xml",
   ]
 
+  # Keep this dependency gated for source tarballs without Git metadata.
   # Only include histograms_xml if we have access to dirmd data, which
   # is used to populate ownership information. This is disabled
   # for non-git checkouts, as dirmd doesn't work on non-git checkout.
