diff --git a/third_party/blink/renderer/modules/webgl/gpu_fingerprint.cc b/third_party/blink/renderer/modules/webgl/gpu_fingerprint.cc
--- /dev/null
+++ b/third_party/blink/renderer/modules/webgl/gpu_fingerprint.cc
@@ -0,0 +1,21 @@
+#include "third_party/blink/renderer/modules/webgl/gpu_fingerprint.h"
+
+#include "components/ungoogled/persona_profile.h"
+
+namespace blink {
+
+std::string GetGLVendorStringForFingerprint() {
+  const auto& persona = ungoogled::CurrentPersona();
+  if (!persona.webgl_real)
+    return persona.webgl_vendor;
+  return "";
+}
+
+std::string GetGLRendererStringForFingerprint(uint64_t /*effective_seed*/) {
+  const auto& persona = ungoogled::CurrentPersona();
+  if (!persona.webgl_real)
+    return persona.webgl_renderer;
+  return "";
+}
+
+}  // namespace blink
