diff --git a/third_party/blink/renderer/core/dom/element.cc b/third_party/blink/renderer/core/dom/element.cc
--- a/third_party/blink/renderer/core/dom/element.cc
+++ b/third_party/blink/renderer/core/dom/element.cc
@@ -7847,6 +7847,14 @@
                                   /*reference_target*/ g_null_atom);
 }
 
+ShadowRoot* Element::ShadowRootForBindings() const {
+  ShadowRoot* root = GetShadowRoot();
+  if (root && root->GetMode() == ShadowRootMode::kClosed &&
+      RuntimeEnabledFeatures::FakeShadowRootEnabled())
+    return root;
+  return OpenShadowRoot();
+}
+
 ShadowRoot* Element::OpenShadowRoot() const {
   ShadowRoot* root = GetShadowRoot();
   return root && root->GetMode() == ShadowRootMode::kOpen ? root : nullptr;
