diff --git a/third_party/blink/renderer/platform/fonts/font_metrics.cc b/third_party/blink/renderer/platform/fonts/font_metrics.cc
--- a/third_party/blink/renderer/platform/fonts/font_metrics.cc
+++ b/third_party/blink/renderer/platform/fonts/font_metrics.cc
@@ -33,6 +33,7 @@
 #include "base/containers/span.h"
 #include "build/build_config.h"
 #include "third_party/blink/renderer/platform/fonts/font_platform_data.h"
+#include "third_party/blink/renderer/platform/fonts/skia/skia_text_metrics.h"
 #include "third_party/blink/renderer/platform/fonts/vdmx_parser.h"
 #include "third_party/blink/renderer/platform/wtf/math_extras.h"
 #include "third_party/skia/include/core/SkFont.h"
@@ -60,6 +61,8 @@
 
   SkFontMetrics metrics;
   font.getMetrics(&metrics);
+  [[maybe_unused]] const bool windows_metrics =
+      SkFontApplyWindowsMetrics(font, &metrics);
 
   if (ascent_override)
     metrics.fAscent = -platform_data.size() * ascent_override.value();
@@ -77,7 +80,7 @@
   static const uint32_t kVdmxTag = SkSetFourByteTag('V', 'D', 'M', 'X');
   int pixel_size = ClampTo<int>(platform_data.size() + 0.5);
   // TODO(xiaochengh): How do we support ascent/descent override with VDMX?
-  if (!ascent_override && !descent_override && !font.isForceAutoHinting() &&
+  if (!windows_metrics && !ascent_override && !descent_override && !font.isForceAutoHinting() &&
       (font.getHinting() == SkFontHinting::kFull ||
        font.getHinting() == SkFontHinting::kNormal)) {
     size_t vdmx_size = face->getTableSize(kVdmxTag);
@@ -123,7 +126,7 @@
     // the descent part of the glyph may be truncated when displayed in a
     // 'overflow: hidden' container.  To avoid that, borrow 1 unit from the
     // ascent when possible.
-    if (descent < metrics.fDescent &&
+    if (!windows_metrics && descent < metrics.fDescent &&
         platform_data.GetFontRenderStyle().use_subpixel_positioning &&
         ascent >= 1) {
       ++descent;
