diff --git a/third_party/blink/renderer/modules/peerconnection/rtc_session_description_request_promise_impl.cc b/third_party/blink/renderer/modules/peerconnection/rtc_session_description_request_promise_impl.cc
--- a/third_party/blink/renderer/modules/peerconnection/rtc_session_description_request_promise_impl.cc
+++ b/third_party/blink/renderer/modules/peerconnection/rtc_session_description_request_promise_impl.cc
@@ -11,6 +11,7 @@
 #include "third_party/blink/renderer/modules/peerconnection/rtc_peer_connection.h"
 #include "third_party/blink/renderer/modules/peerconnection/rtc_session_description.h"
 #include "third_party/blink/renderer/platform/peerconnection/rtc_session_description_platform.h"
+#include "third_party/blink/renderer/platform/peerconnection/fingerprint_ip.h"
 
 namespace blink {
 
@@ -44,6 +45,7 @@
         V8RTCSdpType::Create(platform_session_description->GetType()).value());
     description->setSdp(platform_session_description->Sdp());
     requester_->NoteSdpCreated(*description);
+    description->setSdp(String::FromUtf8(RewriteFingerprintSdp(platform_session_description->Sdp().Utf8())));
     resolver_->Resolve(description);
   } else {
     // This is needed to have the resolver release its internal resources
